src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a user command to list out the tasks currently in the task list.
+ */
+public class ListCommand extends Command {
+    private String userInput;
+
+    public ListCommand(String userInput) {
+        this.userInput = userInput;
+    }
+
+    public boolean isExit() {
+        return false;
+    }
+
+    /**
+     * Lists out the tasks in <code>tasks</code>.
+     *
+     * @param tasks List of <code>Task</code> objects.
+     * @param ui Ui object created by Duke.
+     * @param storage Storage object created by Duke.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        System.out.println("    Here are the tasks in your list:");
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println("    " + (i + 1) + ". " + tasks.get(i));
```

Comment:
Perhaps a constant could be defined in the `Ui` class such that it is easier to print the indentation